### ISSUE TITLE
Description of CONNECT should refer to RFC2817 (Trac #119)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#119]
P2, Section 8.9 ("CONNECT") should probably also point to RFC2817, Section 5.

Migrated from https://trac.ietf.org/ticket/119
```json
{
    "status": "closed",
    "changetime": "2012-03-22T00:07:51",
    "_ts": "1332374871618691",
    "description": "P2, Section 8.9 (\"CONNECT\") should probably also point to RFC2817, Section 5.",
    "reporter": "julian.reschke@gmx.de",
    "cc": "",
    "resolution": "fixed",
    "time": "2008-06-06T13:24:55",
    "component": "p2-semantics",
    "summary": "Description of CONNECT should refer to RFC2817",
    "priority": "normal",
    "keywords": "",
    "milestone": "04",
    "owner": "julian.reschke@gmx.de",
    "type": "editorial",
    "severity": "Active WG Document"
}
```

**Comment by mnot** [comment]
Trac update at 20080707T14:20:32: 
* julian.reschke@gmx.de changed milestone from "unassigned" to "04"
* julian.reschke@gmx.de changed owner from "" to "julian.reschke@gmx.de"

**Comment by mnot** [comment]
Trac update at 20080707T14:29:12: 
* julian.reschke@gmx.de commented:

> Fixed in [274]:
> 
> Resolve #119: refer to RFC2817 for CONNECT method (closes #119).


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "new" to "closed"

**Comment by mnot** [comment]
Trac update at 20120322T00:07:51: 
* mnot@pobox.com changed priority from "" to "normal"
* mnot@pobox.com changed severity from "Candidate WG Document" to "Active WG Document"
